src/elements/content-uploader/__tests__/ItemAction-test.js

```diff
@@ -31,6 +31,14 @@ describe('elements/content-uploader/ItemAction', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should render correctly with STATUS_STAGED', () => {
+        const wrapper = getWrapper({
+            status: STATUS_STAGED,
+        });
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Ideally, unit tests should have explicit assertions in addition to (or in place of) snapshots.